procedures/katello/branch.md.erb

```diff
@@ -25,10 +25,16 @@
 - [ ] Ensure stable Pulp release
   - [ ] Update the [katello-repos](https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/katello/katello-repos/katello-repos.spec#L1-L2)
   - [ ] Update [forklift](https://github.com/theforeman/forklift/blob/master/roles/katello_repositories/defaults/main.yml#L4-L5)
+- [ ] Coordinate with Release Packager that expected changes to associated installer modules are completed.
 
 ## Release Packager
 
 - [ ] Ensure tool_belt config is merged and output from `./tools.rb koji configs/katello/<%= release %>.yaml` matches expectations
+- [ ] Confirm with release owner that all expected changes are present before performing an installer modules release.
```

Comment:
This gives the impression that the Katello Release Packager does these releases but in practice they don't. Until now that has **always** been done by the installer maintainers. I doubt release packagers even have the permissions to do so.